src/lib/stores/nostrocket_state/soft_state/problems.ts

```diff
@@ -331,6 +331,26 @@ function populateChildren(problem: Problem, state: Nostrocket) {
   }
 }
 
+function getDescendants(problem: Problem, state: Nostrocket): Map<String, Problem> {
```

Comment:
This function needs to be recursive so that it searches the entire tree of problems that exist below the current one, not just the next layer down

Reply:
Yeah, I believe it is recursive because using the `queue` it does BFS and iterates through all the children's children